example/spec/parallel/testAPI/ruleSpec.js

```diff
@@ -52,8 +38,70 @@ function isWorkflowTriggeredByRule(taskInput, params) {
   return taskInput.meta.triggerRule && taskInput.meta.triggerRule === params.rule;
 }
 
+describe('When I create a scheduled rule via the Cumulus API', () => {
+  const scheduledRuleName = timestampedName('SchedHelloWorldIntegrationTestRule');
+  const scheduledHelloWorldRule = {
+    name: scheduledRuleName,
+    workflow: 'HelloWorldWorkflow',
+    rule: {
+      type: 'scheduled',
+      value: 'rate(3 minutes)'
+    },
+    meta: {
+      triggerRule: scheduledRuleName
+    }
+  };
+
+  beforeAll(async () => {
+    // Create a scheduled rule
+    await rulesApiTestUtils.postRule({
+      prefix: config.stackName,
+      rule: scheduledHelloWorldRule
+    });
+  });
+
+  describe('the scheduled rule is deleted', () => {
```

Comment:
Before this we should test that the rule got kicked off. Maybe drop the rate down to 1 or 2 minutes.

So it should be like 
```
it('the rule kicks off a workflow')

describe('when the scheduled rule is deleted')

it('does not kick off a workflow')
```